### DISCUSSION TITLE
"Data":"EEEEEE"

### DISCUSSION THREAD

**Discussion by Puppetmaster91** [arendst/Tasmota#14738] [Q&A] [answered]
Hi

I have a question regarding data strings longer than the 24-bit hex.
I have some magnet sensors which broadcast a data key longer than the 24-bit hex which is default in the Sonoff bridge(with tasmota).

I can see the key correctly during the "rfraw 177" command, but question is. Is there any downsides to be in rfraw 177 constantly?
And does anybody have their RF bridge in rfraw 177 mode all the time?

**Comment by sfromis** [comment, marked as answer]
The only downside of staying in `RfRaw 177` is that you do not get the nicer decoding, which may be annoying for other sensors (and maybe more spurious noise). And yes, I do think that others have gotten around the messy situation with outdated RF 433 by staying in raw mode.

**Reply by Puppetmaster91** [reply]
Thank you, for your answer :)
